src/main/java/com/hello/hello/Food.java

```diff
@@ -0,0 +1,14 @@
+package com.hello.hello;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+public class Food {
+    @Getter
+    private String name;
+    @Getter
+    private String materials;
+    @Getter
+    private String country;
```

Comment:
クラス内の全てのフィールドにgetterを作成する場合は、クラスに対して@GetterをつければOKです！
```suggestion
@Getter
public class Food {
    private String name;
    private String materials;
    private String country;
```

Reply:
かしこまりました！
対応いたしました！
47409e8602afceaf744ad8bef710c60f7a060a21